src/components/animations/HourglassAnimation.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import Animated, {
+  Easing,
+  useAnimatedStyle,
+  useDerivedValue,
+  withRepeat,
+  withSequence,
+  withTiming,
+} from 'react-native-reanimated';
+import { Path, Svg } from 'react-native-svg';
+import { Flex } from '../layout';
+import { useColorMode } from '@rainbow-me/design-system';
+import styled from '@rainbow-me/styled-components';
+
+const Container = styled(Flex)({
+  justifyContent: 'center',
+  width: '100%',
+});
+
+const AnimatedPath = Animated.createAnimatedComponent(Path);
+
+const rotationConfig = {
+  duration: 1200,
+  easing: Easing.elastic(1),
+};
+
+const sandConfig = {
+  duration: 1200,
+  easing: Easing.bezier(1, 0.2, 0.47, 0.97),
+};
+
+export default function HourglassAnimation() {
+  const { colorMode } = useColorMode();
+  const darkMode = colorMode !== 'light';
+  const rotateHourglass = useDerivedValue(() => {
+    return withRepeat(
+      withSequence(
+        withTiming(180, rotationConfig),
+        withTiming(0, rotationConfig),
+        withTiming(-180, rotationConfig)
+      ),
+      Infinity,
+      false
```

Comment:
```suggestion
    return withRepeat(
      withSequence(
        withTiming(180, rotationConfig),
        withTiming(0, rotationConfig),
        withTiming(-180, rotationConfig)
      ),
      -1
```
@pugson 
Please, read the documentation here: https://docs.swmansion.com/react-native-reanimated/docs/api/animations/withRepeat/

Instead of `Infinity`, `-1` is intended. No need for `false`. This is a default arg.